ang/crmCaseType.js

```diff
@@ -142,7 +153,13 @@
       'sequence': 'Sequence'
     };
 
-    $scope.caseType = apiCalls.caseType ? apiCalls.caseType : _.cloneDeep(newCaseTypeTemplate);
+    if (isNewCaseType) {
+      $scope.caseType = _.cloneDeep(newCaseTypeTemplate);
```

Comment:
There was an explicit change that checked if this API call exists here 9625aad#diff-1e51710f6a7a59a5c0c0db9a53d71b89R139. Is there a reason why we don't want to check it anymore?

Reply:
The check is still being done:
https://github.com/civicrm/civicrm-core/pull/12098/files#diff-f42508faf29afc0f8ecd02bcb3d6be66R138